### ISSUE TITLE
train loss goes to 0 very quickly (30-50 steps) independent from the model and weights

### ISSUE THREAD

**Issue by onurbarut** [thtrieu/darkflow#512]
UPDATE:

The error was the result of mismatch of the 1 class name in label.txt and xml files.

Issue solved.

Hey,

I don't know what happened, but the training loss goes to 0.00xx very very quickly (30-50 steps), independent of the model and whether pre-trained or not. And there is no prediction box at the end. 

The last thing I did was to type (if relevant):

>> flow xxxx --trainer momentum
then it gave error

>> flow xxxx --trainer adam
it gave error also, then I deleted --trainer flag and later all the trainings had the same issue I wrote above. Did anybody encounter the same issue? 

update: I checked the --trainer adam error, and it was due to that in the checkpoint adam parameters were not found. That's fine because I trained first without --trainer adam flag.

Would this break down whole the code????
